great_expectations/expectations/core/expect_column_distinct_values_to_be_in_set.py

```diff
@@ -193,23 +196,19 @@ def _atomic_prescriptive_template(
                     f"distinct values must belong to this set: {values_string}."
                 )
 
-        if params["row_condition"] is not None:
+        params_with_json_schema: dict = params.dict()
+
+        if params.row_condition:
             (
                 conditional_template_str,
                 conditional_params,
             ) = parse_row_condition_string_pandas_engine(
-                params["row_condition"], with_schema=True
+                params.row_condition.value, with_schema=True
             )
             template_str = f"{conditional_template_str}, then {template_str}"
             params_with_json_schema.update(conditional_params)
 
-        params_with_json_schema = add_values_with_json_schema_from_list_in_params(
-            params=params,
-            params_with_json_schema=params_with_json_schema,
-            param_key_with_list="value_set",
-        )
-
-        return (template_str, params_with_json_schema, styling)
+        return template_str, params_with_json_schema, renderer_configuration.styling
 
     @classmethod
     @renderer(renderer_type=LegacyRendererType.PRESCRIPTIVE)
```

Comment:
This legacy renderer isn't actually leveraging the latest `RendererConfiguration` functionality, but I went ahead and cleaned it up while I was in this file.